database/gorm/to_sql_test.go

```diff
@@ -21,34 +20,33 @@ func TestToSqlTestSuite(t *testing.T) {
 		t.Skip("Skipping tests of using docker")
 	}
 
-	mysqlDocker := NewMysqlDocker(docker.Mysql())
-	query, err := mysqlDocker.New()
-	if err != nil {
-		log.Fatalf("Init mysql error: %s", err)
-	}
+	suite.Run(t, &ToSqlTestSuite{})
+}
 
-	suite.Run(t, &ToSqlTestSuite{
-		query: query,
-	})
+func (s *ToSqlTestSuite) SetupSuite() {
+	postgresQuery := NewTestQuery(docker.Postgres())
```

Comment:
Why are we only testing postgres here, shouldn't we test other driver also?

Reply:
The main purpose is to test if the ToSql function is correct, so one driver is enough.

Reply:
Okay